syft/workers/base.py

```diff
@@ -141,10 +141,10 @@ def load_data(self, data: List[Union[torch.Tensor, AbstractTensor]]) -> None:
 
         """
 
-        for tensor in data:
-
-            self.register_obj(tensor)
-            tensor.owner = self
+        if data:
```

Comment:
Decided to handle a `None` value here and avoid using a {} or [] as default argument value, which is not a good practice overall.